### ISSUE TITLE
In Ghost 5.0 'benefits' has changed to a string

### ISSUE THREAD

**Issue by fueko** [TryGhost/Ghost#14881]
### Issue Summary

Hi! 

Using the syntax:

```
{{#if benefits}}
   {{#foreach benefits}}
      <li>{{{name}}}</li>
   {{/foreach}}
{{/if}}
```

We get an empty list, in version 4.48.x this worked fine, have there been any changes in version 5.0?

### Steps to Reproduce

Use this code:
```
{{#if benefits}}
   {{#foreach benefits}}
      <li>{{{name}}}</li>
   {{/foreach}}
{{/if}}
```
We get an empty list.

### Ghost Version

5.0.0

### Node.js Version

Ghost(Pro)

### How did you install Ghost?

Ghost(Pro)

### Database type

MySQL 8

### Code of Conduct

- [X] I agree to be friendly and polite to people in this repository

**Comment by fueko** [comment]
Hi! @ErisDS Thanks for the information! That is, the attribute `data-members-plan` like `data-members-plan="{{monthly_price.id}}"` also has it been withdrawn? 

Because I see it's not working and in the documentation there is now an option with `data-portal="signup/{{id}}/monthly"` - is this now the only option to connect to Stripe checkout (for custom membership page)?

**Comment by ErisDS** [comment]
We've cleaned up a _lot_ of legacy code and helpers that all focused around single monthly and yearly prices to try to simplify themes now that we have tiers. The newly documented method is correct. The `{{id}` is the id of the tier, and this attribute can be also be fetched from the Portal settings in Admin. 

We're still improving the docs and gscan as fast as we can!

We also added new helpers so you can detect https://ghost.org/docs/themes/helpers/site/#site-feature-flags.

**Comment by fueko** [comment]
Thank you so much for your reply @ErisDS! 
The last question (so if the helper `{{monthly_price.*}}` is withdrawn now) if I want to have separate currencies and prices this usage is correct? 

`{{#if monthly_price}}<sup>{{price currency=currency}}</sup>{{price monthly_price}}{{/if}}`

If I want it in one block:

`{{#if monthly_price}}{{price monthly_price currency=currency}}{{/if}}`

**Comment by ErisDS** [comment]
Yep exactly 🙂

**Comment by ErisDS** [comment]
This is fixed in 5.0.1 along with improved rules for the removal of @price and a new standalone [price helper docs page](https://ghost.org/docs/themes/helpers/price/) with more example code.

Let us know if there's anything else we can clarify or improve
